examples/create_example_data.sql

```diff
@@ -0,0 +1,45 @@
+INSERT INTO "DataSources" ("id","name","type","config","createdAt","updatedAt")VALUES (DEFAULT,'nedb_notes','InMemory','{"options":{"timestampData":true}}','2018-07-02 15:44:45.467 +00:00','2018-07-02 15:44:45.467 +00:00') RETURNING *;
+
+INSERT INTO "GraphQLSchemas" ("id","schema","createdAt","updatedAt") VALUES (DEFAULT,'schema {
+      query: Query
+      mutation: Mutation
+      subscription: Subscription
+    }
+
+    # The query type, represents all of the entry points into our object graph
+    type Query {
+      readNote(id: String): Note
```

Comment:
@aliok I'm just wondering should we change this to `_id`. Same with `updateNote` and `deleteNote`. Having `_id` returned but needing to specify `id` for updates, reads, deletes is a bit weird. What do you think?

Reply:
@darahayes I think it is better if we have the parameter name as `id`. I updated param names for update and delete to `id` in my commit as well.

Reply:
@aliok sounds good to me!